babelsberg/constraintinterpreter.js

```diff
@@ -368,6 +359,50 @@ Object.subclass('Babelsberg', {
         return result;
     },
 
+    chooseConstraintBasedOnMetrics: function(constraints, optimizationPriority) {
+        var minIndex = -1;
+        var constraint = null;
+        if (optimizationPriority === undefined) {
+            optimizationPriority = ['time', 'numberOfChangedVariables'];
```

Comment:
I thought went with "changeDistance" as a name for this metric at some point (as opposed to numeric or squaredChangeDistance for LRueckert's branch) 
It's shorter and more generaly states what is done (what, not how)